ui/spec/integration/controllers/admin_controller_spec.rb

```diff
@@ -0,0 +1,239 @@
+require 'integration_helper'
+
+RSpec.describe AdminController, type: :controller do
+
+  before (:each) do
+    allow(Time).to receive(:now).and_return(Time.now)
```

Comment:
FYI Time.now evaluates in the test every time `Time.now` is called in the controller